mapsims/tests/test_cmb.py

```diff
@@ -0,0 +1,15 @@
+from pixell import enmap
+import numpy as np
+import os,sys
+import healpy as hp
+from mapsims import cmb # FIXME: relative path import?
+
+def test_load_sim():
```

Comment:
Uh oh I just realized this test is guaranteed to fail on Travis, because Travis is not NERSC and doesn't have the lensed alms. @zonca any idea how to mark certain tests not to be run on Travis?

Reply:
On NERSC, the default path to lensed alms works, and one can test the above code with `python setup.py test`.

Reply:
it would be nice to have a small dataset (small ell_max) that can be committed to the repo for testing

Reply:
we would also want to have some small nside output maps to check we get the expected map

Reply:
Ok, that sounds good, I can work on that.

Reply:
see example unit test that loads data embedded in the package: https://github.com/simonsobs/so_pysm_models/commit/2ae587a292ad01eed5443b5375e4facd333a681d

Reply:
This would be resolved if there were a way for mapsims/tests/test_cmb.py to know what the absolute path to the data directory is. Could you fix that?